docs/configuration/configuration.md

```diff
@@ -45,6 +45,7 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | `--custom-templates-dir` | string | path to custom html templates | |
 | `--display-htpasswd-form` | bool | display username / password login form if an htpasswd file is provided | true |
 | `--email-domain` | string \| list  | authenticate emails with the specified domain (may be given multiple times). Use `*` to authenticate any email | |
+| `--errors-to-stdout` | bool | redirects error-level logging to stdout instead | |
```

Comment:
I think this should potentially be more accurately named -- STDOUT (previously STDERR, even though docs said STDOUT -- Good catch!) -- is only the default when `--logging-filename` is set to `""`

Reply:
How about `--errors-to-info-log` maybe?

Reply:
+1 for not including stdout in the flag name

`--errors-to-info-log` seems reasonable to me